src/mcode-pilot/utils/recordToProps.js

```diff
@@ -0,0 +1,142 @@
+import FluxTumorDimensions from '../../dataaccess/mcodev0.1-datasource/model/oncology/FluxTumorDimensions';
+
+import _ from 'lodash';
+
+export default function getProps(patient, condition){
+
+    const tumorMarkers = patient.getMostRecentTumorMarkers(condition);
+
+    const propDict = {
+        // demographics
+        "demographic":
+            {
+                "age":
+                        {
+                            "display": "age",
+                            "valueType": "range",
+                            "range":10,
+                            "value":patient.getAge()
+                        },
+                "diagnosedAge":
+                        {
+                            "display": "age at diagnosis",
+                            "valueType":"range",
+                            "range":10,
+                            "value":patient.getAgeAsOf(new Date(condition.diagnosisDate))
+                        },
+                "race":
+                        {
+                            "display": "race",
+                            "valueType":"string",
+                            "value": patient.patient.race
+                        },
+                "gender":
+                        {
+                            "display":"gender",
+                            "valueType":"string",
+                            "value": _.lowerCase(patient.patient.gender)
+                        }
+            },
+
+        // pathology
+        "pathology":
+            {
+                "grade":
+                        {
+                            "display":"grade",
+                            "valueType":"int",
+                            "value": condition.getMostRecentHistologicalGrade().getGradeAsSimpleNumber()
+                        },
+                "stage":
+                        {
+                            "display":"stage",
+                            "valueType":"string",
+                            "value": safeGet(condition.getMostRecentClinicalStaging(),"stage")
+                        },
+                "size":
+                        {
+                            "display":"size (mm)",
+                            "valueType":"int",
+                            "value":(()=>{
+                                const quantity = safeGet(condition.getMostRecentLabResultOfEachType().find(e=>{return e.constructor.name === FluxTumorDimensions.name}),"quantity")
+                                return safeGet(quantity,"number");
+                        })()
+                }   
+            },
+        "medical history":{
+            "ECOG":
+                {
+                    "display":"ECOG Score",
+                    "valueType":"range",
+                    "range":1,
+                    "value":safeGet(condition.getMostRecentECOGPerformanceStatus(),"value")
+
+                }
+        }
+     
+        
+    }
+
+    if(tumorMarkers){
```

Comment:
Spaces after if and before bracket.

Spaces after parentheses and before bracket